tck/features/clauses/call/Call5.feature

```diff
@@ -29,3 +29,153 @@
 #encoding: utf-8
 
 Feature: Call5 - Results projection
+
+  Scenario: [1] Explicit procedure result projection
+    Given an empty graph
+    And there exists a procedure test.my.proc(in :: INTEGER?) :: (out :: STRING?):
+      | in   | out   |
+      | null | 'nix' |
+    When executing query:
+      """
+      CALL test.my.proc(null) YIELD out
+      RETURN out
+      """
+    Then the result should be, in order:
+      | out   |
+      | 'nix' |
+    And no side effects
+
+  Scenario: [2] Rename procedure call output in RETURN clause
+    Given an empty graph
+    And there exists a procedure test.my.proc(in :: INTEGER?) :: (out :: STRING?):
+      | in   | out   |
+      | null | 'nix' |
+    When executing query:
+      """
+      CALL test.my.proc(null) YIELD out AS a
+      RETURN a
+      """
+    Then the result should be, in order:
+      | a     |
+      | 'nix' |
+    And no side effects
+
```

Comment:
This scenario seems to be wrongly named. If the scenario is changed to actually do the renaming in the `RETURN` clause, it should be moved to Call6. Otherwise and as it is, the scenarios seems to covered by the scenarios outline I suggest in a comment below and should probably removed (if not changed and moved to Call6).

```suggestion
  Scenario: [2] Rename procedure call output in RETURN clause
    Given an empty graph
    And there exists a procedure test.my.proc(in :: INTEGER?) :: (out :: STRING?):
      | in   | out   |
      | null | 'nix' |
    When executing query:
      """
      CALL test.my.proc(null) YIELD out
      RETURN out AS a
      """
    Then the result should be, in order:
      | a     |
      | 'nix' |
    And no side effects

```